book_source/02_demos_tutorials_workflows/02_user_demos/06_Workflow_using_Rmd_template.Rmd/workflow.Rmd

```diff
@@ -0,0 +1,124 @@
+# Introduction
+
+This tutorial provides a step-by-step guide to running the **PEcAn Modular Workflow**. The PEcAn (Predictive Ecosystem Analyzer) system automates ecological modeling, helping researchers analyze plant functional traits, run model simulations, and perform sensitivity analyses.
+
+After setting up PEcAn locally, open **RStudio** in your browser and log in with:
+- **Username**: `carya`
+- **Password**: `illinois`
+
+Then, navigate to `web/workflow.Rmd` and open it to run the workflow. This method eliminates the need to execute the workflow through the web interface once `pecan.xml` is configured.
+
+---
+
+# Prerequisites
+
+Before running this workflow, ensure you have:
+
+- Installed **PEcAn** and its dependencies.
+- An XML settings file (`settings.xml`) configured for your use case.
+- A model binary (e.g., **SIPNET** or **ED2**) specified in your settings.
+
+---
+
+# Load PEcAn Packages
+
+We first load the necessary PEcAn modules required for running the workflow.
+
+```{r libraries, message=FALSE, warning=FALSE}
```

Comment:
add an echo=FALSE so this output doesn't show up in the knit doc. That said, not sure why we want warning=FALSE